pkg/internal/checkup/checkup.go

```diff
@@ -69,8 +70,11 @@ type Checkup struct {
 }
 
 const (
-	VMINamePrefix                 = "dpdk-vmi"
-	TrafficGeneratorPodNamePrefix = "kubevirt-dpdk-checkup-traffic-gen"
+	VMINamePrefix                       = "dpdk-vmi"
+	TrafficGeneratorPodNamePrefix       = "kubevirt-dpdk-checkup-traffic-gen"
+	appLabelKey                         = "app"
```

Comment:
Please consider using a more specific label name, such as `kubevirt-dpdk-checkup/uid`.
It will keep this checkup compatible with the VM latency checkup:
https://github.com/kiagnose/kiagnose/blob/6e3194ad309ab0fab8476da74023e5f7eb0dc7ee/checkups/kubevirt-vm-latency/vmlatency/internal/checkup/checkup.go#L73

Reply:
agree. DONE